### DISCUSSION TITLE
Non Responsive page

### DISCUSSION THREAD

**Discussion by AhmedKAwwad** [community/community#22776] [New to GitHub]
<p>There is an improper appearance for this part of “Follow” button over Search bar<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/5aceb55aaa66f3d5eeb1e0cbe111fe9f80103d53.png" data-download-href="/uploads/short-url/cXjQgMIgb5OnPtoYp5cSFM3zz1N.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/23080326/181073109-b5a58187-d66b-4907-aa77-abb9178732a8.png" alt="image" data-base62-sha1="cXjQgMIgb5OnPtoYp5cSFM3zz1N" width="690" height="329" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/5aceb55aaa66f3d5eeb1e0cbe111fe9f80103d53_2_690x329.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/5aceb55aaa66f3d5eeb1e0cbe111fe9f80103d53_2_1035x493.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/5aceb55aaa66f3d5eeb1e0cbe111fe9f80103d53_2_1380x658.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/5aceb55aaa66f3d5eeb1e0cbe111fe9f80103d53_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1648×788 68.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Check this Github Support , if there is an action needed to report this , inform me How<br>
Thanks in advance</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/ahmedkawwad">@AhmedKAwwad</a>! Thank you for being apart of the community. We’re glad that you shared this example with us as well. Our team has been looking into this and just released some new changes.</p>

**Comment by AhmedKAwwad** [comment]
<p>With my pleasure .you are Welcome.</p>
